index.html

```diff
@@ -1,29 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
     <title>Document</title>
-</head>
-<body>
-    <h1 class="heading">All Awesome Books</h1>
-    <div class="showBooks" id="showBooks"></div>
-    <hr class="hr-line">
-    <div class="container">
+  </head>
+  <body>
+    <header>
+      <nav>
+        <p>Awesome Books</p>
+        <ul class="link">
+          <li><a href="#" class="list-nav">List</a></li>
+          <li>|</li>
+          <li><a href="#" class="books-nav">Add new</a></li>
+          <li>|</li>
+          <li><a href="#" class="contact-nav">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section>
+      <p id="time"></p>
+    </section>
+    <section>
+      <div class="book-list">
+        <h1 class="heading">All Awesome Books</h1>
+        <div class="showBooks" id="showBooks"></div>
+      </div>
```

Comment:
- [ ] I suggest that we improve on this section that displays the list of added books. Our implementation as compared to the given wireframe lacks the alternating background for the list items. Please consider working on this. See screenshots below for comparison:

| Wireframe| Implementation |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/45002654/153281730-c8fe4c64-fe80-4008-b695-6cc749f3d63f.png) | ![image](https://user-images.githubusercontent.com/45002654/153281893-1c5c7d6a-740e-49ec-acc7-5c1ab72ac9d6.png) |

Reply:
Thanks for your review @Lameck1 . We have worked on that and solved it.